core/dictgen/src/rootcling_impl.cxx

```diff
@@ -1334,12 +1334,12 @@ void WriteNamespaceInit(const clang::NamespaceDecl *cl,
 /// array data member.
 /// In case of error, or if the size is not specified, GrabIndex returns 0.
 
-llvm::StringRef GrabIndex(const clang::FieldDecl &member, int printError)
+llvm::StringRef GrabIndex(const cling::Interpreter& interp, const clang::FieldDecl &member, int printError)
```

Comment:
```suggestion
llvm::StringRef GrabIndex(cling::Interpreter &interp, const clang::FieldDecl &member, int printError)
```

Reply:
We cannot remove the `const` without a const_cast in `WriteStreamer` then... Is that what we want?

Reply:
And then we need a `const_cast` or removing of the const in `llvm::StringRef TClingDataMemberInfo::ValidArrayIndex() const`.

Reply:
Sorry, I missed that call. Fine - whatever approach you prefer.